src/clients/patreon/patreonAgent.ts

```diff
@@ -15,60 +8,64 @@ export interface PatreonUserTokens {
 
 export interface PatronFullDetails {
   patreonUserId: string;
-  patronMemberDetails: {
-    lastChargeDate: Date;
-    lastChargeStatus: string;
-    nextChargeDate: Date;
-    currentlyEntitledAmountCents: number;
-  };
+  isMember: Boolean;
```

Comment:
Let's get rid of this guy and instead have the controller throw an error if any of the below are invalid (i.e. null/undefined)